weaviate/collections/classes/config.py

```diff
@@ -1651,19 +1647,13 @@ def sharding(
                 The number of virtual shards per physical shard.
             `desired_count`
                 The desired number of physical shards.
-            `actual_count`
-                The actual number of physical shards.
```

Comment:
this would be a breaking change, Should we leave them in and just have them not do anything and issue a deprecation-warning?